src/main/java/org/folio/dataexp/controllers/JobProfileController.java

```diff
@@ -115,11 +116,20 @@ public ResponseEntity<Void> putJobProfile(UUID jobProfileId, JobProfile jobProfi
     userInfo.setUserName(user.getUsername());
     jobProfile.setUserInfo(userInfo);
 
-    var metadata = jobProfile.getMetadata();
-    metadata.updatedDate(new Date());
-    metadata.updatedByUserId(userId);
-    metadata.updatedByUsername(user.getUsername());
+    var metadataOfExistingJobProfile = jobProfileEntity.getJobProfile().getMetadata();
```

Comment:
There is no guarantee that metadata is present, please add additional null check here.